t/90h2olog.t

```diff
@@ -64,6 +64,9 @@ subtest "h2olog", sub {
 
   my @events = map { decode_json($_) } split /\n/, $trace;
   is scalar(grep { $_->{type} && $_->{tid} && $_->{seq} } @events), scalar(@events), "each event has type, tid and seq";
+
+  my($h3s_accept) = grep { $_->{type} eq "h3s-accept" } @events;
+  ok is_uuidv4($h3s_accept->{"conn-uuid"}), "h3s-accept has a UUIDv4 field `conn-uuid`"
```

Comment:
Maybe we don't need this, now that we have unit tests for UUID generation. We do not test each value provided through h2olog, do we?

Reply:
I'd like to keep the E2E test in addition to unit tests because (1) it's a key to a log collector that uses h2olog, and (2) it's not trivial to handle a string field in h2olog.

Reply:
Works for me.